### PR TITLE
Feature/40516

### PR DESCRIPTION
**Pull request by Aekal**
dev-40516

### CODE REVIEW

File: Magento_Bundle/templates/catalog/product/view/summary.phtml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+                <?= __('Summary') ?>:
+            </div>
+            <div id="bundle-summary" data-container="product-summary">
+                <ul
+                    class="product-view__summary-list list"
+                    data-mage-init='{"productSummary": []}'>
+                </ul>
+                <script data-template="bundle-summary" type="text/x-magento-template">
+                    <li class="product-view__summary-item list__item">
+                        <span class="product-view__summary-item-title"><%- data._label_ %>:</span>
```
Comment (szafran89) [inline Magento_Bundle/templates/catalog/product/view/summary.phtml]: can you split lines for above code

File: Magento_Bundle/templates/catalog/product/view/summary.phtml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+                        <div data-container="options"></div>
+                    </li>
+                </script>
+                <script data-template="bundle-option" type="text/x-magento-template">
+                    <div><?= __('%1 x %2', '<%- data._quantity_ %>', '<%- data._label_ %>') ?></div>
```
Comment (szafran89) [inline Magento_Bundle/templates/catalog/product/view/summary.phtml]: can you remove div or split lines

File: Magento_Bundle/templates/catalog/product/view/summary.phtml [hunk 3]
```diff
@@ -0,0 +1,40 @@
+<?php $_product = $block->getProduct();?>
+<?php if ($_product->isSaleable() && $block->hasOptions()): ?>
+<div
```
Comment (szafran89) [inline Magento_Bundle/templates/catalog/product/view/summary.phtml]: indentation

File: Magento_Bundle/templates/catalog/product/view/type/bundle.phtml [hunk 4]
```diff
@@ -0,0 +1,13 @@
+<?php $_product = $block->getProduct() ?>
+<?php if ($block->displayProductStockStatus()): ?>
+    <?php if ($_product->isAvailable()): ?>
+        <div class="product-view__stock-status">
+            <?= __('In stock') ?>
+        </div>
+    <?php else: ?>
+        <div class="product-view__stock-status product-view__stock-status--out-of-stock">
```
Comment (szafran89) [inline Magento_Bundle/templates/catalog/product/view/type/bundle.phtml]: to new line `class="product-view__stock-status product-view__stock-status--out-of-stock"`

File: Magento_Bundle/templates/catalog/product/view/type/bundle/option/checkbox.phtml [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+        <?= $block->escapeHtml($_option->getTitle()) ?>
+    </h4>
+    <?php if ($block->showSingle()): ?>
+        <?= $block->getSelectionQtyTitlePrice($_selections[0]) ?>
+        <input
+            type="hidden"
+            class="bundle option"
+            name="bundle_option[<?= $_option->getId() ?>]"
+            value="<?= $_selections[0]->getSelectionId() ?>"
+        />
```
Comment (szafran89) [inline Magento_Bundle/templates/catalog/product/view/type/bundle/option/checkbox.phtml]: you can remove `/`

File: Magento_Bundle/templates/catalog/product/view/type/bundle/option/checkbox.phtml [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+                    class="checkbox__field bundle option"
+                    id="bundle-option-<?= $_option->getId() ?>-<?= $_selection->getSelectionId() ?>"
+                    type="checkbox"
+                    <?php if ($_option->getRequired()) echo 'data-validate="{\'validate-one-required-by-name\':\'input[name^=&quot;bundle_option[' . $_option->getId() . ']&quot;]:checked\'}"'?>
+                    name="bundle_option[<?= $_option->getId() ?>][<?= $_selection->getId() ?>]"
+                    data-selector="bundle_option[<?= $_option->getId() ?>][<?= $_selection->getId() ?>]"
+                    <?php if ($block->isSelected($_selection)) echo ' checked="checked"' ?>
+                    <?php if (!$_selection->isSaleable()) echo ' disabled="disabled"' ?>
+                    value="<?= $_selection->getSelectionId() ?>"
+                />
```
Comment (szafran89) [inline Magento_Bundle/templates/catalog/product/view/type/bundle/option/checkbox.phtml]: you can remove `/`

File: Magento_Bundle/templates/catalog/product/view/type/bundle/option/radio.phtml [hunk 7]
```diff
@@ -0,0 +1,93 @@
...
+                            type="radio"
+                            class="radio__field bundle option"
+                            id="bundle-option-<?= $_option->getId() ?>-<?= $_selection->getSelectionId() ?>"
+                            <?php if ($_option->getRequired()) echo 'data-validate="{\'validate-one-required-by-name\':true}"'?>
+                            name="bundle_option[<?= $_option->getId() ?>]"
+                            data-selector="bundle_option[<?= $_option->getId() ?>]"
+                            <?php if ($block->isSelected($_selection)) echo ' checked="checked"' ?>
+                            <?php if (!$_selection->isSaleable()) echo ' disabled="disabled"' ?>
+                            value="<?= $_selection->getSelectionId() ?>"
+                        />
```
Comment (szafran89) [inline Magento_Bundle/templates/catalog/product/view/type/bundle/option/radio.phtml]: you can remove `/`

File: Magento_Bundle/templates/catalog/product/view/summary.phtml (hunk 1 above)
Comment (Aekal) [reply Magento_Bundle/templates/catalog/product/view/summary.phtml]: Done

File: Magento_Bundle/templates/catalog/product/view/summary.phtml (hunk 2 above)
Comment (Aekal) [reply Magento_Bundle/templates/catalog/product/view/summary.phtml]: Done

File: Magento_Bundle/templates/catalog/product/view/summary.phtml (hunk 3 above)
Comment (Aekal) [reply Magento_Bundle/templates/catalog/product/view/summary.phtml]: Done

File: Magento_Bundle/templates/catalog/product/view/type/bundle.phtml (hunk 4 above)
Comment (Aekal) [reply Magento_Bundle/templates/catalog/product/view/type/bundle.phtml]: Moved

File: Magento_Bundle/templates/catalog/product/view/type/bundle/option/checkbox.phtml (hunk 5 above)
Comment (Aekal) [reply Magento_Bundle/templates/catalog/product/view/type/bundle/option/checkbox.phtml]: Removed

File: Magento_Bundle/templates/catalog/product/view/type/bundle/option/checkbox.phtml (hunk 6 above)
Comment (Aekal) [reply Magento_Bundle/templates/catalog/product/view/type/bundle/option/checkbox.phtml]: Removed

File: Magento_Bundle/templates/catalog/product/view/type/bundle/option/radio.phtml (hunk 7 above)
Comment (Aekal) [reply Magento_Bundle/templates/catalog/product/view/type/bundle/option/radio.phtml]: Removed

File: Magento_Bundle/templates/catalog/product/view/type/bundle/options.phtml [hunk 8]
```diff
@@ -0,0 +1,39 @@
...
+                <?php endif; ?>
+                <?= $block->getOptionHtml($option) ?>
+            <?php endforeach; ?>
+        </div>
+    <?php else: ?>
+        <p>
+            <?= __('No options of this product are available.') ?>
+        </p>
+    <?php endif; ?>
+<?php endif;?>
```
Comment (szafran89) [inline Magento_Bundle/templates/catalog/product/view/type/bundle/options.phtml]: missing space

Comment (Aekal) [reply Magento_Bundle/templates/catalog/product/view/type/bundle/options.phtml]: Added

**Pull request merged**